spec/models/manageiq/providers/hawkular/middleware_manager/refresher_spec.rb

```diff
@@ -3,15 +3,15 @@
     allow(MiqServer).to receive(:my_zone).and_return("default")
     auth = AuthToken.new(:name => "test", :auth_key => "valid-token", :userid => "jdoe", :password => "password")
     @ems_hawkular = FactoryGirl.create(:ems_hawkular,
-                                       :hostname        => 'localhost',
-                                       :port            => 8080,
+                                       :hostname        => 'hservices.torii.gva.redhat.com',
+                                       :port            => 80,
                                        :authentications => [auth])
   end
 
   it "will perform a full refresh on localhost" do
     VCR.use_cassette(described_class.name.underscore.to_s,
                      :allow_unused_http_interactions => true,
-                     :decode_compressed_response     => true) do # , :record => :new_episodes) do
+                     :decode_compressed_response     => true, :record => :new_episodes) do
```

Comment:
I think for the final version you want to remove the `:record => :new_episodes` part